TSC 2.6 reports error in LL1Analyzer.ts

New error when built w/ TypeScript 2.6:

```
src/atn/LL1Analyzer.ts(253,10): error TS2358: The left-hand side of an 'instanceof' expression must be of type 'any', an object type or a type parameter.
```
It seems that the variable `t` at this line has a type of `never`, which seems like a bug in TypeScript!   

It looks like this has given us trouble before because an earlier line has a cast in it.   I've inserted a more robust workaround.

--- a/package.json
+++ b/package.json
@@ -58,7 +58,7 @@
     "source-map-support": "^0.4.3",
     "std-mocks": "^1.0.1",
     "typedoc": "^0.5.1",
-    "typescript": "^2.4.1"
+    "typescript": "^2.6.2"
   },
   "dependencies": {}
 }

--- a/src/atn/LL1Analyzer.ts
+++ b/src/atn/LL1Analyzer.ts
@@ -248,7 +248,8 @@ export class LL1Analyzer {
 			}
 			else {
 //				System.out.println("adding "+ t);
-				let set: IntervalSet | undefined = (<Transition>t).label;
+				t = <Transition>t;
+				let set: IntervalSet | undefined = t.label;
 				if (set != null) {
 					if (t instanceof NotSetTransition) {
 						set = set.complement(IntervalSet.of(Token.MIN_USER_TOKEN_TYPE, this.atn.maxTokenType));

